src/main/java/com/puppycrawl/tools/checkstyle/utils/ScopeUtil.java

```diff
@@ -44,16 +44,20 @@ private ScopeUtil() {
      */
     public static Scope getDeclaredScopeFromMods(DetailAST aMods) {
         Scope result = null;
-        for (DetailAST token = aMods.getFirstChild(); token != null;
+        for (DetailAST token = aMods.getFirstChild(); token != null && result == null;
```

Comment:
I am more just curious abut this change. Is it just to exit the loop faster? I am surprised pitest doesn't flag this in some way.

Reply:
Utils doesn't have `REMOVE_CONDITIONALS` active. When it does, this will most likely come up again.

Reply:
Yes this is for fast exit from the loop in cases like `private static final`. No reason to go through the other modifiers if we already met `public` or `private`.